requirements2.txt

```diff
@@ -1,4 +0,0 @@
-# Python2 needs the concurrent.futures backport.
-# We handle this logic in setup.py but we need
-# a separate file to track this in our requirements file.
-futures==2.2.0
```

Comment:
So this gets handled by setup.py now? It does not to be included in the requirements.txt?